src/core/thread/network_data_leader_ftd.cpp

```diff
@@ -354,7 +354,7 @@ void Leader::SendCommissioningGetResponse(const Coap::Message &   aRequest,
             {
                 if (cur->GetType() == aTlvs[index])
                 {
-                    SuccessOrExit(error = message->Append(cur, sizeof(NetworkDataTlv) + cur->GetLength()));
+                    SuccessOrExit(error = message->AppendTlv(*cur));
```

Comment:
Adding a note since this one looks different than all other changes.

Note that `cur` is  a `MeshCop::Tlv` pointer which inherits from `ot::Tlv`

The `sizeof(NetworkDataTlv)` here refers to `ot::NetworkData::NetworkDataTlv` (we are in the `ot::NetworkData` namespace) which has same size as `ot:Tlv`.